fix: quickstart demos

SkypLabs opened the pull request:
Hi there,

I started to play with your quickstart examples and I realised that the Kratos configuration files are targeting very old versions of the tool (e.g. `v0.7.1-alpha.1`), which make them invalid according to your current configuration JSON schema.

This PR upgrades the version numbers in the Kratos configuration files located in `contrib/quickstart`. I also took the liberty to upgrade the Docker image of Oathkeeper in `quickstart-oathkeeper.yml`.

## Related issue(s)

N/A

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

Some of the checklist items are irrelevant for this PR.

## Further Comments

None.

SkypLabs commented:
The Oathkeeper quickstart demo is partially broken as the `/sessions` path is not part of access rules despite the presence of a link to the "Session Information" page on the Dashboard:

![Screenshot 2022-12-07 at 17-23-45 Session Information](https://user-images.githubusercontent.com/9932586/206234318-080f53a1-ae9d-43c3-8926-871ff1e9a9a4.png)

Would you like me to fix it in this PR as well? If so, I could rename it to "fix: quickstart demos" or similar.

aeneasr commented:
> Would you like me to fix it in this PR as well? If so, I could rename it to "fix: quickstart demos" or similar.

That would be awesome! :)

SkypLabs commented:
> The Oathkeeper quickstart demo is partially broken as the /sessions path is not part of access rules despite the presence of a link to the "Session Information" page on the Dashboard

I removed the unused routes (`/debug` and `/dashboard`) and added the missing ones (`/sessions`, `/health/alive` and `/health/ready`).

I also rebased my branch onto `master`.

aeneasr commented:
Thank you very much!
